src/Traits/Incrementable.php

```diff
@@ -2,6 +2,9 @@
 
 namespace PHLAK\SemVer\Traits;
 
+/**
+ * Trait for incrementing a version object.
+ */
```

Comment:
```suggestion
```

Reply:
I think it’s bad practice not to PHPDoc (public) classes, traits, interfaces, functions. You have to decide. 🙃